sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -267,7 +133,7 @@ def test_show_stats_and_model_version(self, client):
 
         self.assertIsNotNone(response)
         self.assertIsNotNone(response.model_version)
-        self.assertEqual("2021-01-11", response.model_version)
+        self.assertIsNotNone(response.model_version)
```

Comment:
Duplicate line

Reply:
oh wait, this actually isn't a duplicated line, I just replaced `self.assertEqual("2021-01-11", response.model_version)` with `self.assertIsNotNone(response.model_version)`

Reply:
nvm i'm actually going to go with that specific model version, since we actually pass the model version in during our call